inference-engine/tests/ie_test_utils/functional_test_utils/CMakeLists.txt

```diff
@@ -4,24 +4,33 @@
 
 set(TARGET_NAME funcTestUtils)
 
-list(APPEND EXPORT_DEPENDENCIES inference_engine_lp_transformations
-    commonTestUtils inference_engine inference_engine_legacy)
+list(APPEND EXPORT_DEPENDENCIES
+        inference_engine_lp_transformations
+        commonTestUtils
+        inference_engine
+        inference_engine_legacy
+        ngraphFunctions
+        inference_engine_transformations
+        )
 
 addIeTarget(
     NAME ${TARGET_NAME}
     TYPE STATIC
     ROOT ${CMAKE_CURRENT_SOURCE_DIR}
     ADD_CPPLINT
     DEVELOPER_PACKAGE
-    LINK_LIBRARIES ngraphFunctions
```

Comment:
Why did you add `ngraphFunctions` library to public depenndencies?

Reply:
Good point, thanks, fixed)